scripts/load-test-remix.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Simulate what Thimble does when a project is remixed. This is a simplification
```

Comment:
can we add this to the `program.` chain below in a function call to `description()`?

Kinda like https://github.com/mozilla/publish.webmaker.org/blob/master/scripts/export.js#L98